test/components/legendTests.ts

```diff
@@ -254,6 +254,34 @@ describe("Legend", () => {
     svg.remove();
   });
 
+  it("accept formatter and show entity labels correcly", () => {
```

Comment:
how about
`it("can set which formatter to use to change how entry labels are displayed"...`?